helm/0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+# Helm Job-Executor Integration
+
+This integration shows you how to leverage [Job-executor-service](https://github.com/keptn-contrib/job-executor-service) for deployment with `helm` on your Kubernetes cluster.
```

Comment:
Are you saying that an integration can be just documentation?  That seems like a confusing use of the term "integration".

Reply:
Yes, that's what I am saying here. And we already have plenty of those (everything related to webhook-service and job-executor), so I'm not inventing the words here. Example: https://artifacthub.io/packages/keptn/keptn-integrations/githubaction